src/gui/vector/qgsvectorlayerproperties.cpp

```diff
@@ -1263,11 +1261,25 @@ void QgsVectorLayerProperties::saveStyleAs()
           return;
         }
 
-        mLayer->saveStyleToDatabase( dbSettings.name, dbSettings.description, dbSettings.isDefault, dbSettings.uiFileContent, msgError );
+        mLayer->saveStyleToDatabase( dbSettings.name, dbSettings.description, dbSettings.isDefault, dbSettings.uiFileContent, errorMessage, dlg.styleCategories() );
 
-        if ( !msgError.isNull() )
+        if ( !errorMessage.isNull() )
+        {
+          mMessageBar->pushMessage( infoWindowTitle, errorMessage, Qgis::MessageLevel::Warning );
+        }
+        else
+        {
+          mMessageBar->pushMessage( infoWindowTitle, tr( "Style saved" ), Qgis::MessageLevel::Success );
+        }
+        break;
+      }
+      case LOCAL:
+      {
+        QString infoWindowTitle = tr( "Save default style to local database" );
```

Comment:
Title case

Reply:
It's not a window title, but I kept the same variable name then the case just above.
It's the "title" for the message bar. All other same strings above are written with this normal case style.